eng/pipelines/common/templates/wasm-library-tests.yml

```diff
@@ -32,11 +32,18 @@ jobs:
         value: ${{ parameters.alwaysRun }}
       - name: allWasmContainsChange
         value: $[ dependencies.evaluate_paths.outputs['SetPathVars_allwasm.containsChange'] ]
+      - name: _tryRunSmokeOnlyArg
+        value: ${{ parameters.tryRunSmokeOnly }}
+      - name: _runSmokeOnlyToUse
```

Comment:
I don't have a solution in mind but this feels like going a little too far, we now have `_tryRunSmokeOnlyArg`, `_runSmokeOnlyToUse`, `_runSmokeTestsOnlyArg` and `runSmokeOnlyArg` variables/parameters which all do subtly different things.

I'd love if we could come up with a way to simplify this.